py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/s3_utils.py

```diff
@@ -98,16 +106,21 @@ def _move_s3_object(filename: str, destination: str) -> None:
         logging.debug("copy key 1: %s", copy_key)
 
         # json args for cop
-        copy_source = {
-            "Bucket": source,
-            "Key": copy_key,
-        }
+        destination_bucket = s3_resource.Bucket(destination)
```

Comment:
use resource api rather than client one. the concept of

`move = copy + delete` remains.